app/src/pages/inside/logsPage/logItemInfo/logItemInfoTabs/attachments/attachments.scss

```diff
@@ -0,0 +1,73 @@
+.attachments-wrap {
+  width: 100%;
+  text-align: center;
+
+  .log-attachments {
+    margin: 0 auto;
+    width: 80%;
+    &.hideMainArea :global .carousel-slider {
+      display: none;
+    }
+  
+    .preview-container {
+      line-height: 365px;
+      cursor: pointer;
+  
+      .preview-image {
```

Comment:
No need to make such deep nesting in this case. And we don't need to reflect HTML structure in CSS rules in most cases. All class names are unique.
The only case for nesting is when we need to make some rules more specific.

Reply:
@DiscoElevator I had to keep the `global` tree since we need to override the default style, and I didn't wanted to use `!important` in a top level class.